ci/docker/python-wheel-windows-test-vs2019.dockerfile

```
@@ -54,7 +54,7 @@ RUN (if "%python%"=="3.9" setx PYTHON_VERSION "3.9.13") & \
     (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.11") & \
     (if "%python%"=="3.11" setx PYTHON_VERSION "3.11.9") & \
     (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.5") & \
```

Not strictly required but we can update to the latest on chocolatey.
```suggestion
    (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.6") & \
```